modules/telco-ran-gitops-operator-and-ztp-plugins.adoc

```diff
@@ -12,7 +13,10 @@ New in this release::
 Description::
 {gitops-shortname} and {ztp} plugins provide a {gitops-shortname}-based infrastructure for managing cluster deployment and configuration.
 Cluster definitions and configurations are maintained as a declarative state in Git.
-ZTP plugins provide support for generating installation CRs from the `SiteConfig` CR and automatic wrapping of configuration CRs in policies based on `PolicyGenTemplate` CRs.
+You can apply `ClusterInstance` CRs to the hub cluster where the `SiteConfig` Operator renders them as installation CRs.
+Alternatively, you can use the {ztp} plugin to generate installation CRs directly from `SiteConfig` CRs.
+The {ztp} plugin supports automatic wrapping of `Configuration` CRs in policies based on `PolicyGenTemplate` CRs.
+
 +
 You can deploy and manage multiple versions of {product-title} on managed clusters using the baseline reference configuration CRs.
```

Comment:
Can we augment this to include a couple things
- Supporting multiple versions is possible when the source CRs and policy (PolicyGenTemplate or PolicyGenerator) are all stored in git, allowing creation of per-version policies simultaneously
- We recommend keeping reference CRs and custom CRs under different directories. This allows a patch/update of the reference CRs by simple replacement of all directory contents without disturbing the custom CRs content.